client/my-sites/upgrades/domain-management/edit/mapped-domain.jsx

```diff
@@ -29,9 +29,12 @@ const MappedDomain = React.createClass( {
 			);
 		}
 
+		const expirationString = domain.expirationMoment && domain.expirationMoment.format( 'MMMM D, YYYY' ) ||
+			this.translate( 'Never', { context: 'Expiration time for a mapped domain' } );
```

Comment:
I'd change this to `<em>{ this.translate( 'Never Expires' ) }</em>` for consistency sake with what's shown for `*.wordpress.com` domains in the [domain's details page](https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/upgrades/domain-management/edit/wpcom-domain.jsx#L35) and [purchase page](https://github.com/Automattic/wp-calypso/blob/master/client/me/purchases/list/item/index.jsx#L85).